src/components/proofs/MobileProofCategories.tsx

```diff
@@ -21,6 +21,8 @@ export default function ({
       onChange={(selectedValue) => {
         setCategory(selectedValue as CategoriesTitles)
       }}
+      colorfulCurrentValue
+      hideAfterMd
```

Comment:
I guess we should do it without a prop. It just unnecessary

Reply:
We need it to keep things both on mobile and desktops, removing it means:
1) Make a default `hideAfterMd = true`
2) Add `hideAfterMd={false}` in other places

So let's keep it, better have a `true` prop, than `prop={false}`

Reply:
I mean that you need it only in one place, so you can wrap this dropdown into `div` with a class that will display on certain screens. You don't need to pass this prop to Dropdown.